Package.resolved

```diff
@@ -65,12 +65,12 @@
         }
       },
       {
-        "package": "xcodeproj",
-        "repositoryURL": "https://github.com/tuist/xcodeproj.git",
+        "package": "XcodeProj",
+        "repositoryURL": "https://github.com/mrylmz/xcodeproj.git",
```

Comment:
This is fine temporarily, but we should have a PR on the project explaining the fix before this can be merged.

Reply:
[👍](https://github.com/tuist/xcodeproj/pull/410)